doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -16,7 +16,7 @@ A dataset contains three levels of metadata:
 
 #. **Citation Metadata**: any metadata that would be needed for generating a data citation and other general metadata that could be applied to any dataset;
 #. **Domain Specific Metadata**: with specific support currently for Social Science, Life Science, Geospatial, and Astronomy datasets; and
-#. **File-level Metadata**: varies depending on the type of data file - see *File Handling and Uploading* section below for more details). 
+#. **File-level Metadata**: varies depending on the type of data file - for more details see :ref:`File Handling <file-handling>` section below).
```

Comment:
It's always great when we link like this rather than just saying "it's down there somewhere." Good work.

Reply:
My plan (as discussed in the sprint planning meeting) was to mention that Data Explorer provides the option to subset data, but after a quick review of the docs I'm not going to do so because:

- This issue has been around for quite awhile and we haven't received much (any?) feedback about it being broken, so I'm not sure how much it's used
- If we start to mention specific things that are available in external tools, it becomes a documentation maintenance burden for us as those tools can change at any time

I acknowledge that we should revisit and (dare I say) modularize the docs related to external tools in the future, but I'm OK not doing that now. I'll move this one on to QA. Thanks for the discussion here.